android/src/main/res/layout/fragment_metrics_drawer.xml

```diff
@@ -296,15 +296,15 @@
 
             <Button
                 android:id="@+id/button_change_power_setting"
-                style="?android:attr/buttonStyleSmall"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
+                android:background="@layout/button_selector"
                 android:layout_above="@+id/map_info_title"
                 android:layout_alignEnd="@+id/cell_plus_wifi_layout"
                 android:layout_alignRight="@+id/cell_plus_wifi_layout"
-                android:layout_marginBottom="4dp"
                 android:minHeight="0dp"
-                android:text="@string/change_power_saving" />
+                android:text="@string/change_power_saving"
+                android:layout_marginBottom="10dp" />
```

Comment:
This needed to change for KitKat from 4dp to 10dp.   I've tested on Android 4.4 and 5.1 to verify that vertical text justification is consistent.